src/usr/local/www/diag_authentication.php

```diff
@@ -40,7 +40,7 @@
 		$input_errors[] =  sprintf(gettext('%s is not a valid authentication server'), $_POST['authmode']);
 	}
 
-	if (empty($_POST['username']) || empty($_POST['password'])) {
```

Comment:
Why change this here? It would alter existing behavior for non-LDAP logins unrelated to this change.

Reply:
Hi,

Yes, but this is okay. this page is only used for debugging/diags. I believe it is better for this page to give an auth result which matches the return value of *_backed().